src/output/kmlcreator.cc

```diff
@@ -491,7 +491,32 @@ void KMLCreator::writePlanePlacemarkElement(QXmlStreamWriter &writer, Placemark
 
     int index = 0;
     foreach(GPSRecord c, p->mPoints) {
+        std::string week_ms_str = c.msec().toStdString();
+        const char *tschar = week_ms_str.c_str();
+        char* endptr;
+        uint32_t week_ms = strtoul(tschar, &endptr, 10);
```

Comment:
fetching the week_ms can be done much easier: `int weekMS = c.msec().toInt();` for example.